manifests/o/Oven-sh/Bun/1.1.2/Oven-sh.Bun.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.6.0.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Oven-sh.Bun
+PackageVersion: 1.1.2
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: bun-windows-x64\bun.exe
+  PortableCommandAlias: bun
+- RelativeFilePath: .\bun-windows-x64\bun.exe

```

I think it should be the same?
```suggestion
- RelativeFilePath: bun-windows-x64\bun.exe

```